presto-main/src/main/java/com/facebook/presto/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -269,7 +283,7 @@ else if (matches.size() > 1) {
                 RowType rowType = checkType(field.getType(), RowType.class, "field.getType()");
                 Type rowFieldType = null;
                 for (RowField rowField : rowType.getFields()) {
-                    if (rowField.getName().equals(node.getName().getSuffix())) {
+                    if (rowField.getName().get().equals(node.getName().getSuffix())) {
```

Comment:
Maybe allow for unnamed row field here.  It shouldn't happen, but a future bug might cause a hard to read exception.